olad/www/Makefile.mk

```diff
@@ -33,3 +44,42 @@ dist_www_DATA = \
     olad/www/vertical.gif \
     olad/www/wand.png \
     olad/www/warning.png
+dist_new_DATA = \
+    olad/www/new/index.html
+dist_views_DATA = \
+    olad/www/new/views/plugins.html \
+    olad/www/new/views/plugin-info.html \
+    olad/www/new/views/overview.html \
+    olad/www/new/views/universes.html \
+    olad/www/new/views/universe-add.html \
+    olad/www/new/views/universe-overview.html \
+    olad/www/new/views/universe-keypad.html \
+    olad/www/new/views/universe-patch.html \
+    olad/www/new/views/universe-settings.html \
+    olad/www/new/views/universe-sliders.html \
+    olad/www/new/views/universe-rdm.html
+dist_js_DATA = \
+    olad/www/new/js/app.min.js
```

Comment:
The non-minified version should probably be in EXTRA_DIST via javascript/new-src.